index.html

```diff
@@ -14,7 +14,7 @@
       <div id = "lviv">
         <img src="images/wall2.jpg " width="920px">
       </div>
-      <div id = "kruivka">
+      <div id = "kruivka" data-width = "49.841293" data-longitude = "24.03227">
```

Comment:
Hey @Vovan707, what do you mean with `width`? Is it a latitude? If yes, then please rename.